src/main/java/sharadhr/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,40 @@
+package sharadhr.duke.command;
+
+import sharadhr.duke.exception.DukeInvalidArgumentException;
+import sharadhr.duke.io.Output;
+import sharadhr.duke.io.Storage;
+import sharadhr.duke.task.Task;
+import sharadhr.duke.task.TaskList;
+
+/**
+ * DoneCommand
+ */
+public class DoneCommand extends Command {
+    private int position;
+
+    DoneCommand(String[] commandTokens, int position) throws DukeInvalidArgumentException {
+
+        // if (this.argumentTokens.length != 1 || !this.argumentTokens[0].matches("\\d+")) {
```

Comment:
Don't retain commented code. If necessary you can retrieve it from version control later